tests/test_decode.py

```diff
@@ -1,7 +1,22 @@
 """Test cases for the chardetng_py module."""
-from chardetng_py import decode
+import pytest
+from chardetng_py import detect
 
 
 def test_decode() -> None:
     """Very basic decode litmus test."""
-    assert decode(b"Jakby r\xeaka Boga") == "Jakby rêka Boga"
+    raw = b"Jakby r\xeaka Boga"
+    assert raw.decode(detect(raw)) == "Jakby rêka Boga"
```

Comment:
There is no `decode` function, so I switched this to `detect`. I think the goal here to test round-tripping, but if not and I should change this, let me know.

Reply:
That makes sense. This binding largely wraps the rust library, so it was more meant to be a smoke test.